api/classes/publisher.js

```diff
@@ -195,7 +195,7 @@ function createOrUpdatePublishedProject() {
     title: project.title,
     tags: project.tags,
     description: project.description,
-    date_updated: (new Date()).toISOString()
+    date_updated: new Date()
```

Comment:
this will use `toString()`, which is not necessarily the same as `toISOString()` - better to keep it explicitly ISO?